elasticsearch-model/test/integration/mongoid_import_test.rb

```diff
@@ -0,0 +1,155 @@
+require 'test_helper'
+
+begin
+  require 'mongoid'
+  session = Moped::Connection.new("localhost", 27017, 0.5)
+  session.connect
+  ENV["MONGODB_AVAILABLE"] = 'yes'
+rescue LoadError, Moped::Errors::ConnectionFailure => e
+  $stderr.puts "MongoDB not installed or running: #{e}"
+end
+
+if ENV["MONGODB_AVAILABLE"]
+  $stderr.puts "Mongoid #{Mongoid::VERSION}", '-'*80
+
+  logger = ::Logger.new($stderr)
+  logger.formatter = lambda { |s, d, p, m| " #{m.ansi(:faint, :cyan)}\n" }
+  logger.level = ::Logger::DEBUG
+
+  Mongoid.logger = logger unless ENV['QUIET']
+  Moped.logger   = logger unless ENV['QUIET']
+
+  Mongoid.connect_to 'mongoid_articles'
+
+  if defined?(BSON::ObjectId)
```

Comment:
Hmm -- why fiddle with `BSON::ObjectId` at all? This is not something people do in their app probably? Can we make it work in a different way?